ui/src/views/404/Footer/index.jsx

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+// This is an example of styles that are defined inline
+const style = {
+    borderTopWidth: '3px',
+    borderColor: '#0061A6',
+    borderTopStyle: 'solid',
+    paddingTop: '2px',
+    fontSize: '11px',
+    lineHeight: '14px',
+    margin: '15px 0',
+    clear: 'both',
+};
+
+const item = ({ text, ...props }) =>
+    <li
+        style={{
+            display: 'inline',
+            ':after': {
+                content: '" | "',
+            },
+        }}>
+        <a
+            {...props}
+            style={{
+                color: '#005689',
+                textDecoration: 'none',
+            }}>
+            {text}
+        </a>
+    </li>;
+
+const items = [
+    { href: '/help', text: 'Help' },
+    { href: '/help/contact-us', text: 'Contact us' },
+    {
+        href: 'https://www.surveymonkey.com/s/theguardian-beta-feedback',
+        text: 'Feedback',
+        target: '_blank',
+    },
+    { href: '/help/terms-of-service', text: 'Terms &amp; conditions' },
+    { href: '/help/privacy-policy', text: 'Privacy policy' },
+];
+
+export default () =>
```

Comment:
Can we give exported JSX stateless components a name? If we don't do this, I seem to remember that the component shows up as `<Unknown>` in React dev tools. Perhaps we can also add a lint rule for this one day.